### DISCUSSION TITLE
Can anyone help implment the basic example?

### DISCUSSION THREAD

**Discussion by ianpaschal** [clauderic/dnd-kit#1577] [Q&A]
I'm hoping to reproduce [this Storybook story](https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/docs/core-droppable-usedroppable--multiple-droppables) for a project of mine. Ideally with all the animations and CSS transitions.

I went in circles all day yesterday trying to recreate it from scratch based on the documentation, then trying to reverse engineer the Storybook code, then getting frustrated, turning back to the documentation and trying to start from scratch again...

I'm hopeful that maybe some of the many people who use DnD-kit (it seems to be the most popular these days) could share the basics needed to get these animations working?

**Comment by yf-yang** [comment]
Why not check its source code directly?
https://github.com/clauderic/dnd-kit/blob/e9215e820798459ae036896fce7fd9a6fe855772/stories/1%20-%20Core/Droppable/Droppable.story.tsx#L95

**Reply by ianpaschal** [reply]
> **trying to reverse engineer the Storybook code**, then getting frustrated, turning back to the documentation and trying to start from scratch again...

The Storybook examples are not modular enough to be taken as examples. Instead they're crazy-over-optimized and not even close to real-world implementations.
